pages/orgs/[orgId]/settings.tsx

```diff
@@ -112,7 +115,9 @@ export default function EditOrgPage() {
       setLoading(false);
     };
 
-    fetchUserSubscriptions();
+    if (parseBooleanEnvValue(window.__env.NEXT_PUBLIC_STRIPE_ENABLED)) {
```

Comment:
Why do you access the window.__env directly instead of using loadConfig which uses it to configure the configuration values?